projects/hslayers/components/layer-manager/widgets/idw-widget.component.ts

```diff
@@ -81,7 +81,7 @@ export class HsIdwWidgetComponent
   }
 
   listNumericAttributes(features: Feature[]): string[] {
-    return this.hsLayerUtilsService.listNumericAttributes(features);
+    return this.hsLayerUtilsService.listAttributes(features, true);
```

Comment:
generally I don't like these function signatures, as they are hard to read. Adding a boolean switch as another argument is obscuring the real behaviour of the function call.
I propose either to change the definition of `listAttributes()` to 
`listAttributes(features: Feature[], {numericOnly = false} = {})`
or revive `listNumericAttributes` as a wrapper like
```
listNumericAttributes(features) {
  this.listAttributes(features, true)
}
```

Reply:
Thansk for a suggestion I agree. 
For the two I kinda like the wrapper tbh.
And also, since we are planning on releasing new major version I might take a look at such a functions all around the code base. Will make a separate PR that would be based on this branch.

Reply:
That would be an awesome refactoring, thx!

Reply:
Turned out that we dont actually have much of these :D There are function which could use `options` instead of more params but thats a bit different Id say. So just [6ee86ea](https://github.com/hslayers/hslayers-ng/pull/5146/commits/6ee86ea1e8269bec04bec44ad90a40f614a76d04) after all